Source/effects.cpp

```diff
@@ -4,7 +4,7 @@
 
 int sfxdelay; // weak
 int sfxdnum;
-void *sfx_stream;
+HANDLE sfx_stream;

```

Comment:
This should be safe to upstream. Please submit a PR for https://github.com/diasurgical/devilution